src/setupTests.js

```diff
@@ -17,14 +15,9 @@ window.ResizeObserver = ResizeObserver;
   });
 });
 
-// Reset zustand stores before each test
-// https://github.com/pmndrs/zustand/issues/242#issuecomment-729004730
-const initialLineConfig = useLineConfig.getState();
-const initialHeatmapConfig = useHeatmapConfig.getState();
-
-beforeEach(() => {
-  useLineConfig.setState(initialLineConfig, true);
-  useHeatmapConfig.setState(initialHeatmapConfig, true);
+// Clear localStorage after each test
+afterEach(() => {
+  window.localStorage.clear();
 });
```

Comment:
We no longer need to reset the stores themselves between tests; only `localStorage` like in `app.spec.ts`!